webpack/environments/devConfig.js

```diff
@@ -0,0 +1,72 @@
+const chalk = require("chalk")
+// @ts-check
```

Comment:
Was this comment being on line 2 intentional?

Reply:
Yes, I think I would need to install `@types/chalk` to get it to stop erroring -- doesn't seem worth it for the small amount we use it at the moment.